### PR TITLE
MultiversX march ecosystem updates

### PR DESCRIPTION
**Pull request by schimih**

### CODE REVIEW

File: data/ecosystems/m/multiversx-elrond.toml [hunk 1]
```diff
@@ -1239,3 +1239,54 @@ url = "https://github.com/yum0e/my-simple-explanations"
...
+url = "https://github.com/esdt-space/mx-dapp-template"
+
+[[repo]]
+url = "https://github.com/Moonlorian/vesting-dapp"
+
+[[repo]]
+url = "https://github.com/cyberdeckstudios/WalletConnectV2_Demo"
+
+[[repo]]
+url = "https://github.com/cyberdeckstudios/pyWalletConnect"
```
Comment (eherrerosj) [inline data/ecosystems/m/multiversx-elrond.toml:1292]: can you please point out how can I identify MultiversX within the code of `pyWalletConnect`?

**Review by eherrerosj — CHANGES_REQUESTED** [review]

File: data/ecosystems/m/multiversx-elrond.toml (hunk 1 above)
Comment (schimih) [reply data/ecosystems/m/multiversx-elrond.toml:1586]: Solved

File: data/ecosystems/m/multiversx-elrond.toml [hunk 2]
```diff
@@ -1239,3 +1239,54 @@ url = "https://github.com/yum0e/my-simple-explanations"
...
+url = "https://github.com/adrianlari/the-bucharest-hackathon"
+
+[[repo]]
+url = "https://github.com/cmalex23/hackathon2023/"
+
+[[repo]]
+url = "https://github.com/CoinDrip-finance/vesto-sc"
+
+[[repo]]
+url = "https://github.com/Awesome-Purple-Hackers"
```
[the remark this answers is not in the document]
Comment (schimih) [reply data/ecosystems/m/multiversx-elrond.toml]: Done. Please check! :pray:

**Comment by schimih** [comment]
@eherrerosj could you please check one more time?
I want to bring the updates for april and I would like to know which branch I should use as base.
Thanks.

File: data/ecosystems/m/multiversx-elrond.toml (hunk 1 above)
Comment (eherrerosj) [reply data/ecosystems/m/multiversx-elrond.toml:1586]: hey @schimih, I'm somehow still seeing the pyWalletConnect repo, did you remove it from the list?

**Comment by eherrerosj** [comment]
> @eherrerosj could you please check one more time? I want to bring the updates for april and I would like to know which branch I should use as base. Thanks.

awesome, I will make sure we have these for the next update (ETA mid-May). Adding a couple more comments

File: data/ecosystems/m/multiversx-elrond.toml (hunk 1 above)
Comment (eherrerosj) [reply data/ecosystems/m/multiversx-elrond.toml:1586]: @schimih any chance you can take a look at this so we can finalize this PR?

**Review by eherrerosj — APPROVED** [review]

**Pull request merged**